doc/connecting.md

```diff
@@ -19,9 +19,13 @@ myst:
     property=og:locale: "en_US"
 ---
 
-# Connecting
+# Connecting to Databases with SQL Magic
 
-Connection strings are [SQLAlchemy URL](http://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls) standard.
+Learn how to connect to various databases using SQL Magic in this tutorial. SQL Magic is a Jupyter Notebook extension that allows you to execute SQL queries directly in your notebook cells. We'll show you how to establish connections, connect securely, and use existing `sqlalchemy.engine.Engine` instances.
+
+## Establishing a connection
+
+Connection strings follow the SQLAlchemy URL standard. Here are some example connection strings for various databases:
```

Comment:
@idomic  can you please point me to the issue so I have an idea of how to word this change?

Reply:
Added minimal wording about this in here https://github.com/ploomber/jupysql/pull/385/commits/9031bc6efec9a39148ccea63fffeb7b4cdf0c091 as it looks like this feature is work in progress. 

@yafimvo  please comment in this thread on whether the wording below would be correct


### Custom Connection Support

JupySQL's next goal is to provide support for custom connections that are not directly supported by SQLAlchemy. This allows users to run JupySQL functionality, such as `%sql`, `%sqlcmd`, `%sqlplot`, `%sqlrender`, and the ggplot API, without causing disruptions to the existing API or registering a new dialect.

**Note** This feature is still under development, and improvements are being made to provide more seamless integration with custom database connections.

Reply:
@lfunderburk  Yes, that sounds correct. I guess we won't support `%sqlcmd` actions at first (since we currently rely on SQLAlchemy to get this information).

Reply:
ok, @idomic  please re-review and let me know of any issues.

Reply:
removed paragraph after more discussion 
closing this comment as it was removed
https://github.com/ploomber/jupysql/pull/385/commits/45986532693e64bc58aa21dc5a5c9a352a7a5899